src/TraceEvent/TraceEvent.cs

```diff
@@ -1383,6 +1383,16 @@ public virtual unsafe TraceEvent Clone()
             }
             return ret;
         }
+
+        public virtual TraceEvent CloneToTemplate()
```

Comment:
Does this need to be public?  If not, my preference is to keep this detail internal.

Reply:
My mistake. As I was around the `Clone` I used the same access modifiers.